client/src/components/PhotoUpload.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useContext, useEffect } from 'react';
+import Button from '@material-ui/core/Button';
+import FormData from 'form-data';
+import axios from 'axios';
+import { Typography } from '@material-ui/core';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import AuthContext from '../state_management/AuthContext';
+
+const loadingImage = 'https://team-watermelon-bigdeal-images.s3.amazonaws.com/1594749857168';
```

Comment:
Just for the future (and also now if it doesn't take too long), you can use something like Material-UI's Progress component (https://material-ui.com/components/progress/) So that you have full control over the loading animation/colours/etc. and it doesn't have hit a different website to download